app/code/Magento/GiftMessage/Model/Plugin/OrderItemGet.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\GiftMessage\Model\Plugin;
```

Comment:
Please put it to `Magento\GiftMessage\Plugin`

Reply:
All Plugins are currently in  Magento\GiftMessage\Model\Plugin it make not sense